app/models/job_proxy_dispatcher.rb

```diff
@@ -193,12 +195,11 @@ def self.waiting?
   end
 
   def pending_jobs(target_class = VmOrTemplate)
-    class_name = target_class.base_class.name
```

Comment:
For both `ContainerImage` and `VmOrTemplate` the result was their class name ("ContainerImage", "VmOrTemplate" as strings).
https://github.com/ManageIQ/manageiq/pull/15519#issuecomment-313335655